databases/week2/homework.sql

```diff
@@ -0,0 +1,119 @@
+-- Active: 1738236443102@@0.0.0.0@3307@library_database
+SELECT * from task
+
+-----Add a task with these attributes: title, description, created, updated, due_date, status_id, user_id---
```

Comment:
Thank you for the comments; it made it a lot easier to review. It would be nice to more clearly separate the different parts of the homework, e.g., by putting them into different files. 🙂

A tiny nitpick is that these comments are technically illegal in MySQL as the double dash for a comment must be followed by whitespace, i.e.:
- Illegal: `--My comment`
- Legal: `-- My comment`